### ISSUE TITLE
Problem: documentation for Timespec

### ISSUE THREAD

**Issue by leejw51crypto** [crypto-com/chain#1640]
Timespec is UNIX EPOCH, which is 4 bytes, and expire in 2038 year
but we're not unix epoch, which is 8 bytes,
also specify unit `seconds` in comments

**Comment by tomtau** [comment]
@leejw51 no, time spec is u64 (8 bytes) -- so expires in the year 292277026596
